packages/components/src/components/textarea/textarea.examples.md

```diff
@@ -0,0 +1,183 @@
+reader users the corresponding information:
+
+1. Add a unique ID to the `slot="label"` element
+1. Add a unique ID to the `slot="message"` element (if they are created)
+1. Add corresponding `aria-labelledby="some-label-id"` to the `textarea` element which points to the `label` ID
+1. Add corresponding `aria-describedby="some-description-id some-message-id"` to the `textarea` element which points to
+   both, the `description` ID (if set) and the `message` ID when the (error/success) message appears
+
+## Custom styling
+
+The `p-textarea` component allows for customization through CSS Custom Properties (also known as CSS Variables). These
+variables enable you to override certain styles of the native textarea element within the Shadow DOM, such as adjusting
+the `min-height` and `resize` properties.
+
+```scss
+// default CSS variables
+--p-textarea-min-height: 200px;
+--p-textarea-resize: vertical;
+
+// overwrite with CSS variables
+p-textarea {
+  --p-textarea-min-height: 100px;
+  --p-textarea-resize: horizontal;
+}
+```
+
+<script lang="ts">
+import Vue from 'vue';  
+import {getTextareaCodeSamples} from "shared/src";
+import Component from 'vue-class-component';
+import { FORM_STATES } from '../../utils'; 
+
+@Component
+export default class Code extends Vue {
+  config = { themeable: true, spacing: 'block' };
+
+  get theme(): Theme {
+    return this.$store.getters.playgroundTheme;
+  }
+
+  formExample = getTextareaCodeSamples('default');
+  controlledExample = getTextareaCodeSamples('example-controlled');
+
+  get labelMarkup() {
+    return `<p-textarea name="some-name" label="Some label"></p-textarea>`;
+  }
+
+  withDescriptionText =
+`<p-textarea name="some-name" label="Some label" description="Some description"></p-textarea>`;
+
+  counter =
+`<p-textarea name="some-name" label="Some label" value="Some value" max-length="200"></p-textarea>
+<p-textarea name="some-name" label="Some label" value="Some value" max-length="200" show-counter="false"></p-textarea>`;
+
+  state = 'error';
+  states = FORM_STATES;
+  get stateMarkup() {
+    const attr = `message="${this.state !== 'none' ? `Some ${this.state} validation message.` : ''}"`;
+    return `<p-textarea aria-invalid="${this.state === 'error'}" name="some-name" label="Some label" value="Some value" state="${this.state}" ${attr}></p-textarea>`;
+    }
+    
+  slots = `<p-textarea name="some-name" aria-labelledby="some-label-id" aria-describedby="some-description-id some-message-id" state="error">
+  <span slot="label" id="some-label-id">Some label with a <a href="https://designsystem.porsche.com">link</a>.</span>
+  <span slot="description" id="some-description-id">Some description with a <a href="https://designsystem.porsche.com">link</a>.</span>
+  <span slot="message" id="some-message-id">Some error message with a <a href="https://designsystem.porsche.com">link</a>.</span>
+</p-textarea>`;
```

Comment:
Because we handle all the ARIA relationship inside of the component, we don't need to set all those extra ARIA attributes on the host element anymore. And in consequence, you can also remove the ID attributes from the Slots  as well.

Reply:
updated